### DISCUSSION TITLE
Is it expected that FluentWait not available in Appium 8 ?

### DISCUSSION THREAD

**Discussion by DzmitryIlyin** [appium/java-client#1638] [Q&A]
Hi,
It seems that Appium 8 beta/beta-2 doesn't have org.openqa.selenium.support.ui package thus for e.g. FluentWait is not available. Is it expected/bug/some additional setting required?

**Comment by MoOoG** [comment]
I'm also interested in this. Would be great to get some documentation what can be used...

**Comment by valfirst** [comment]
Please, check see the resolution here: https://github.com/appium/java-client/issues/1700#issuecomment-1142168191

**Reply by MoOoG** [reply]
Thanks, that did it. I added org.seleniumhq.selenium:selenium-support dependency.
